quiz/endpoint-2-results.php

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>QuizFlow Design</title>
```

Comment:
I'm not commenting on this for something for you to change, but so that you might catch this later if I forget.

These pages will be SEO crawlable and visible in the browser menu, so these titles shouldn't be QuizFlow related. For the regular question pages, we'll use the title of the quiz itself, and for the endpoints pages we'll concatenate the names of the products and some other flavor text. I have to echo this all from the DB so it can't be done right now anyway, but just keep this in mind in case you find that I forgot to implement it.